core/src/main/resources/error/error-classes.json

```diff
@@ -166,6 +172,12 @@
     "message" : [ "The second argument of '%s' function needs to be an integer." ],
     "sqlState" : "22023"
   },
+  "UDF_CLASS_IMPLEMENT_MULTI_UDF_INTERFACE" : {
```

Comment:
Could you make it shorter like `MULTI_UDF_INTERFACE_ERROR`. We are going to use error classes in our public doc to refer to errors. For readability, it would be nice to have not so long names.

Reply:
Thanks for explain. Will fix.